lib/util/pssh.js

```diff
@@ -220,22 +221,16 @@ shaka.util.Pssh = class {
     const systemId = shaka.util.Pssh.defaultSystemIds_.get(keySystem);
     const psshInfo = new shaka.util.Pssh(initData, true);
 
-    // If there is only a single pssh, return the original array.
-    if (psshInfo.dataBoundaries.length <= 1) {
-      return initData;
-    }
-
     const index = psshInfo.systemIds.indexOf(systemId);
     // If the index is -1 we can not found the correct pssh
     if (index === -1) {
       return initData;
```

Comment:
Should this perhaps return null if the system ID can't be found?  (You would then also need to make the return type nullable by removing the ! before Uint8Array.)

Reply:
It's not possible return a nullable due other restrictions but it's possible return a empty array. I'll change it.